TestProjects/UniversalGraphicsTest_2D/Assets/Test/Runtime/Renderer2DTests.cs

```diff
@@ -1,121 +0,0 @@
-using NUnit.Framework;
-using UnityEngine;
-using UnityEngine.Rendering.Universal;
-using UnityEngine.Rendering.Universal;
-
-[TestFixture]
-class Renderer2DTests
```

Comment:
This is removed from _UniversalGraphicsTest_2D_ but is not added to the new package.

Is that expected?

Reply:
No, this is a mistake. Good catch!